### PR TITLE
Fix a bug on initializing benchmark results in JsonOutput

### PR DESCRIPTION
**Pull request by tfujiwar**
`self.bench` was always empty.

### CODE REVIEW

**Comment by tfujiwar** [comment]
@tkng Could you check as an owner and readability reviewer, please?
@goe Could you give LGTM if this is OK?

File: output_template/python/lmnet/utils/output.py [hunk 1]
```diff
@@ -33,13 +33,13 @@ class JsonOutput():
...
     """
 
-    def __init__(self, task, classes, image_size, data_format, bench={}):
+    def __init__(self, task, classes, image_size, data_format, bench=None):
         assert task in Tasks
         self.task = task
         self.classes = classes
         self.image_size = image_size
         self.data_format = data_format
-        self.bench = {}
```
Comment (iizukak) [inline output_template/python/lmnet/utils/output.py]: Wow, funny code :)

**Comment by tfujiwar** [comment]
@tkng Could you check Python readability?

**Pull request merged**